hub/apps/how-tos/uno-multiplatform.md

```diff
@@ -45,75 +45,101 @@ Now that you've created your first WinUI 3 app, you might be wondering how to re
+1. Create a new C# solution using the **Uno Platform App** type from Visual Studio's **Start Page**. You may need to choose a different name for your solution.
+
+2. Choose a base template to take your Hello World application multi-platform
 
     :::image type="content" source="images/hello-world/vsix-new-project-options.png" alt-text="Uno solution template for project startup type":::
 
+    While the **Blank** template could be used for simplicity, we normally recommend using the **Default** template for production-ready code. The **Default** template already includes a series of [features](https://platform.uno/docs/articles/external/uno.extensions/doc/ExtensionsOverview.html#learn-about-unoextensions-features) and good practices which will jumpstart your application development.
+
 3. You can optionally choose to customize your app based on the sections on the left side:
-    - **Framework** allows to choose which `TargetFramework` your app will use. [.NET 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) is a commonly appropriate choice.
+    - **Framework** allows to choose which `TargetFramework` your app will use. [.NET 7.0](https://dotnet.microsoft.com/download/dotnet/7.0) is a commonly appropriate choice.
     - **Platforms** provides a list of platforms your application will support. You can choose to support all platforms, or a subset of them
-    - **Presentation** gives a choice about using MVVM (e.g. [MVVM Toolkit](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/mvvm/)) or Uno Platform's MVUX and Feeds
+    - **Presentation** gives a choice about using MVVM (e.g. [MVVM Toolkit](/dotnet/communitytoolkit/mvvm/)) or Uno Platform's MVUX and Feeds
     - **Projects** gives the ability to add a Server project for APIs and hosting for the WebAssembly project
     - **Testing** provides Unit Testing and [UI Testing projects](https://github.com/unoplatform/Uno.UITest)
     - **Features** provides support for WebAssembly PWA and optional VS Code support files
-    - **Extensions** allows to choose for additional [Uno.Extensions](https://github.com/unoplatform/uno.extensions) to kickstart your app faster
+    - **Extensions** allows to choose for additional [Uno.Extensions](https://github.com/unoplatform/uno.extensions) to kickstart your app faster (described above)
     - **Application** sets the App ID for relevant platforms, used when publishing on various app stores.
     - **Theme** gives the ability to change between Fluent and Material themes
 
-4. Click the create button
+4. Click the **Create** button
 
 5. Wait for the projects to be created, and their dependencies to be restored
 
 6. A banner at the top of the editor may ask to reload projects, click **Reload projects**:
     :::image type="content" source="images/hello-world/vs2022-project-reload.png" alt-text="Visual Studio banner offering to reload your projects to complete changes":::
 
```

Comment:
End this secion the same way as the Winui hello world tutorial does:

After creating your project, you should see the following default file structure in your Solution Explorer:

:::image type="content" source="images/hello-world/collapsed-file-structure.png" alt-text="Default file structure":::